rust_src/src/frames.rs

```diff
@@ -507,13 +507,13 @@ pub fn next_frame(frame: LispFrameOrSelected, miniframe: LispObject) -> LispFram
     // a valid candidate will be returned regardless of its position.
     while passed < 2 {
         for_each_frame!(f => {
```

Comment:
This change and the one above it do not belong and will conflict with other active PRs that are resolving the situation. Please remove them from this PR.